nativescript-core/data/observable/observable.ts

```diff
@@ -57,6 +57,22 @@ export class Observable implements ObservableDefinition {
         this.notifyPropertyChange(name, newValue, oldValue);
     }
 
+    public setProperty(name: string, value: any) {
+        const oldValue = this[name];
+        if (this[name] === value) {
+            return;
+        }
+        this[name] = value;
+        this.notifyPropertyChange(name, value, oldValue);
+
+        const specificPropertyChangeEventName = name + "Change";
+        if(this.hasListeners(specificPropertyChangeEventName)) {
```

Comment:
you need to add a whitespace bellow:
```suggestion
        if (this.hasListeners(specificPropertyChangeEventName)) {
```